### ISSUE TITLE
Assistive Technology (AT) Updates

### ISSUE THREAD

**Issue by ckenst** [ckenst/kenst.com#49]
Start with these:

- Change "Latest Articles" to H1 from H3 `class="archive-title"` + add an `aria-label`
- Email input field should have a `aria-label="newsletter signup"`
- Site Navigation should have a `aria-label="Main Navigation"`
- Article page navigation should have a `aria-label="article page navigation"`
- Hero Profile should have a `aria-label="Profile with photo and slogan"`
- Article names should have `aria-label` with the name of the blog post

Found using https://chrome.google.com/webstore/detail/siteimprove-accessibility/efcfolpjihicnikpmhnmphjhhpiclljc?hl=en-US
